ic/managed/verifier/verity_verifier.did

```diff
@@ -1,38 +1,22 @@
-type ProofVerificationResponse = vec ProofResponse;
-
-type ProofResponse = variant {
-	SessionProof : text;
-	FullProof : text;
+type Environment = variant { Production; Development; Staging };
+type Payload = record { sent : text; received : text };
+type PayloadBatch = record { payloads : vec Payload; notary_pub_key : blob };
+type PresentationBatch = record {
+  presentations : vec text;
+  notary_pub_key : blob;
 };
-
-type MerkleTree = record {
-	nodes : vec text;
-	num_leaves : nat64;
-	root : text;
+type PublicKeyReply = record { sec1_pk : text; etherum_pk : text };
+type Result = variant { Ok : vec PayloadBatch; Err : text };
+type Result_1 = variant { Ok : VerificationResponse; Err : text };
+type VerificationResponse = record {
+  signature : text;
+  root : text;
+  payload_batches : vec PayloadBatch;
 };
-
-type DirectVerificationResponse = record {
-	results : ProofVerificationResponse;
-	root : text;
-	signature : text;
-};
-
-type DirectVerificationResult = variant {
-	Ok : DirectVerificationResponse;
-	Err : text;
-};
-
-type ProofBatch = record {
-  proof_requests : vec text;
-  notary_pub_key : text;
-};
-
-
-service : {
-	"ping" : () -> (text) query;
-	"verify_proof_direct" : (proof_requests : vec text, notary_pub_key : text) -> (DirectVerificationResult);
```

Comment:
Can we reintroduce these same handlers - eg. `verify_proof_direct` to simply proxy the `verify_direct` ?
This way we offer backward compatibility and do not need to update all the integrations.

Support required for:
1. `verify_proof_direct`
2. `verify_proof_async`
3. `verify_proof_direct_batch`
4. `verify_proof_async_batch`

Reply:
Very fair point…

In that case, we run both canisters in parallel.

Reply:
I'm currently checking where is the develop branch. It looks like it is far away from the main branch.